packages/api-client/src/views/Request/RequestSection/RequestAuth/RequestAuth.vue

```diff
@@ -2,28 +2,57 @@
 import ViewLayoutCollapse from '@/components/ViewLayout/ViewLayoutCollapse.vue'
 import { useWorkspace } from '@/store'
 import { useActiveEntities } from '@/store/active-entities'
+import {
+  ADD_AUTH_OPTIONS,
+  type SecuritySchemeGroup,
+  type SecuritySchemeOption,
+} from '@/views/Request/consts'
 import { displaySchemeFormatter } from '@/views/Request/libs'
-import { type Icon, ScalarIcon } from '@scalar/components'
+import {
+  type Icon,
+  ScalarButton,
+  type ScalarButton as ScalarButtonType,
+  ScalarComboboxMultiselect,
+  ScalarIcon,
+  useModal,
+} from '@scalar/components'
 import { isDefined } from '@scalar/oas-utils/helpers'
-import { computed } from 'vue'
+import { computed, ref } from 'vue'
 
+import DeleteRequestAuthModal from './DeleteRequestAuthModal.vue'
 import RequestAuthDataTable from './RequestAuthDataTable.vue'
 
-const { selectedSecuritySchemeUids } = defineProps<{
+const { selectedSecuritySchemeUids, layout = 'client' } = defineProps<{
   selectedSecuritySchemeUids: string[]
   title: string
+  layout?: 'client' | 'reference'
 }>()
 
+const emit = defineEmits<{
+  'update:selectedSecuritySchemeUids': [string[]]
+}>()
 const { activeCollection, activeRequest } = useActiveEntities()
-const { securitySchemes } = useWorkspace()
+const {
+  isReadOnly,
+  securitySchemes,
+  securitySchemeMutators,
+  requestMutators,
+  collectionMutators,
+} = useWorkspace()
+
+const comboboxButtonRef = ref<typeof ScalarButtonType | null>(null)
+const deleteSchemeModal = useModal()
+const selectedScheme = ref<{ id: string; label: string } | undefined>(undefined)
```

Comment:
```suggestion
const selectedScheme = ref<{ id: string; label: string } | null>(null)
```
Usually we prefer `null` for explicitly empty values over `undefined` undefined would be more like non-existent than empty

Reply:
✔️